### PR TITLE
Define qcow image name in constant, and update to 20171124.

### PR DESCRIPTION
**Pull request by mjeveritt**

### CODE REVIEW

File: files/qemu_check.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ def command(cmd, timeout_sec):
 
 isfile = os.path.isfile(vmimage)
 if not isfile:
-    proc2 = subprocess.Popen('wget -N http://gentoo.osuosl.org/experimental/amd64/openstack/gentoo-openstack-amd64-defa\
-ult-20170521.qcow2 -O '+ vmimage, stdout=subprocess.PIPE, shell=True)
+    proc2 = subprocess.Popen('wget -N http://gentoo.osuosl.org/experimental/amd64/openstack/' +
```
Comment (kuzetsa) [inline files/qemu_check.py]: (*indent in the usual python-y way*)

```
proc2 = subprocess.Popen(cmd_wget, stdout=subprocess.PIPE, shell=True)
```

File: files/qemu_check.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 d.close()
 
 vmimage = '/tmp/gentoo.qcow2'
-
+openstack = "gentoo-openstack-amd64-default-20171124"
```
Comment (kuzetsa) [inline files/qemu_check.py]: I don't actually write python, so let's call this pseudocode:

```
BaseURI = 'http://gentoo.osuosl.org/experimental/amd64/openstack/gentoo-openstack-amd64-default-'
SnapshotDate = '20171124'
ImageURI = BaseURI + SnapshotDate + '.qcow2'
cmd_wget = 'wget -N ' + ImageURI + '-O ' + vmimage
```

File: files/qemu_check.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 d.close()
 
 vmimage = '/tmp/gentoo.qcow2'
-
```
Comment (aliceinwire) [inline files/qemu_check.py]: pep8:
expected 2 blank lines, found 1 [E302]

**Comment by mjeveritt** [comment]
Can we add pep8 to the repo checks, anyone?

I'll push these updates imminently, see if I can get a pep8 check locally....

**Comment by aliceinwire** [comment]
would be nice to add pep8 as repo checks.

File: files/qemu_check.py [hunk 4]
```diff
@@ -20,11 +20,12 @@ def kill_proc(proc, timeout):
     timeout["value"] = True
     proc.kill()
 
+
 def command(cmd, timeout_sec):
     work = False
     timeout = {"value": False}
     proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
-    #kill_proc = lambda p: p.kill()
+    #  kill_proc = lambda p: p.kill()
```
Comment (aliceinwire) [inline files/qemu_check.py]: don't leave previously commented commands, just remove it.

File: files/qemu_check.py [hunk 5]
```diff
@@ -36,7 +37,7 @@ def command(cmd, timeout_sec):
                 break
     finally:
         timer.cancel()
-    #proc.kill()
+    #  proc.kill()
```
Comment (aliceinwire) [inline files/qemu_check.py]: same here

**Comment by mjeveritt** [comment]
Ok so thats just the qcow image updates now...

**Comment by aliceinwire** [comment]
comment is now:
Define qcow image name as constant, and update.

File: files/qemu_check.py [hunk 6]
```diff
@@ -48,7 +52,10 @@ def command(cmd, timeout_sec):
 
 for vmlinuz in vmlinuz_list:
     print(vmlinuz)
-    work = command('qemu-system-x86_64 -m 128M -kernel /boot/vmlinuz-'+ vmlinuz +' -nographic -serial mon:stdio -hda '+ vmimage +' -append "root=/dev/sda1 console=ttyS0,115200n8 console=tty0"', 120)
+    cmd_qemu = 'qemu-system-x86_64 -m 128M -kernel /boot/vmlinuz-' + vmlinuz + \
+               ' -nographic -serial mon:stdio -hda ' + vmimage + \
+               ' -append "root=/dev/sda1 console=ttyS0,115200n8 console=tty0"'
+    work = command(cmd_qemu, 120)
```
Comment (aliceinwire) [inline files/qemu_check.py]: 120 could be a constant
tiimeout_sec = 120

**Comment by aliceinwire** [comment]
thanks

**Pull request merged**